### PR TITLE
dock_with_cube parameter name mapping

### PR DESCRIPTION
**Pull request by Spirou42**
Fixing parameter name mapping for approach_angle and distance_from_marker in dock_with_cube

### CODE REVIEW

**Comment by Spirou42** [comment]
sorry have created the branch from the wrong merge point so the pull request also contains my last fix.

File: anki_vector/opengl/opengl_vector.py [hunk 1]
```diff
@@ -550,9 +550,14 @@ def __init__(self, custom_object, is_fixed: bool):
             super().__init__(custom_object)
 
         self.is_fixed = is_fixed
-        self.x_size_mm = custom_object.archetype.x_size_mm
```
Comment (msintov) [inline anki_vector/opengl/opengl_vector.py]: Let's revert changes to this file. Thanks!

File: examples/apps/3d_viewer/3d_viewer.py [hunk 2]
```diff
@@ -33,7 +33,8 @@ def main():
                            show_3d_viewer=True,
                            enable_face_detection=True,
                            enable_custom_object_detection=True,
-                           enable_nav_map_feed=True):
+                           enable_nav_map_feed=True,
```
Comment (msintov) [inline examples/apps/3d_viewer/3d_viewer.py]: Let's also revert changes to this file.

**Comment by msintov** [comment]
@Spirou42 Please complete a Contributor License Agreement - see https://github.com/anki/vector-python-sdk/blob/master/CONTRIBUTING.md . We're unable to accept code until we've received this. (Note: You only need to complete this step once, you don't need to complete it for every Pull Request.)

Thank you for your PR!

**Comment by Spirou42** [comment]
I already have.

**Comment by msintov** [comment]
@Spirou42 I confirm you've completed the CLA, thank you!

**Pull request closed without merging**